packages/fxa-payments-server/public/locales/en-US/main.ftl

```diff
@@ -397,3 +397,5 @@ new-user-submit = Subscribe Now
 
 manage-pocket-title = Looking for your { -brand-name-pocket } premium subscription?
 manage-pocket-body = To manage it, <a>click here</a>.
+
+payment-method-header = { $prefix }Choose your payment method
```

Comment:
Probably should leave a comment to explain what `{ $prefix }` is to the localizers.

Reply:
And maybe have the space after the prefix here instead of on the prefix itself?

Reply:
I split this out into 1 translation with prefix and one without. The react code then decides which to render depending on the prefix value.